docs/rest_api.md

```diff
@@ -192,6 +200,14 @@ operation to be preformed. Each nic may have no more than one pending
 network operation; an attempt to queue a second action will result in an
 error.
 
+Authorization requirements:
+
+* Access to the project to which `<node>` is assigned.
+* If `<network>` is not public, access to the project named by the network's
+  `"access"` field is required.
+
```

Comment:
If the network isn't public, then it has to be in the same project as the node (we currently check this).

Maybe something like:
* Node must be in same project as network, if network isn't public.

That could also be placed in `Possible errors:`